src/js/localStorageFunctions.js

```diff
@@ -0,0 +1,5 @@
+const storeListItemData = (listData) => {
+  localStorage.setItem('items', JSON.stringify(listData));
+};
+
```

Comment:
Have you read about the KISS method of code? To Keep It Simple and Smart, I would suggest you do implicit return since you are using an arrow function and it is returning just one line. You can take of the opening and closing curly braces and move the localStorage call to the top to be in one line. It'll make your code clean and simple. 👍

Reply:
yeah you are right  thanks bro

Reply:
👍